packages/editor/src/lib/editor/Editor.ts

```diff
@@ -890,6 +890,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 */
 	undo(): this {
 		this._flushEventsForTick(0)
+		this.complete()
 		this.history.undo()
 		return this
 	}
```

Comment:
@steveruizok can you think of any reason we wouldn't want to stop any in-progress interactions on regular undo/redo? It seems right to me, and it naturally produces correct behaviours with the cropping stuff.

Reply:
No, this seems correct to me too.